WeakAuras/WeakAuras.lua

```diff
@@ -4833,6 +4861,15 @@ function WeakAuras.PerformActions(data, type, region)
     end
     wipe(region.active_glows_hidefunc)
   end
+  if type == "finish" and region.unitframe_monitor then
+    local id = region.id .. region.cloneId or ""
```

Comment:
This can be problematic if you have:
a) id: ABC, cloneId D
b) id: A, cloneId BCD

which have the same id.